inz-health-data/Issues/nhi/description.xml

```diff
@@ -0,0 +1,34 @@
+<IssueSpecification version="4">
+    <Name>
+        <en>Demo INZ NHI Number</en>
+        <nl>Demo INZ NHI Number</nl>
+    </Name>
+    <ShortName>
+        <en>NHI Number</en>
+        <nl>NHI Number</nl>
+    </ShortName>
+    <SchemeManager>irma-demo</SchemeManager>
+    <IssuerID>inz-health-data</IssuerID>
+    <CredentialID>nhi</CredentialID>
+    <Description>
+        <en>Demo New Zealand National Health Identification</en>
+        <nl>Demo New Zealand National Health Identification</nl>
+    </Description>
+    <ShouldBeSingleton>true</ShouldBeSingleton>
+    <IssueURL>
+        <en>https://demo.irma.nz</en>
+        <nl>https://demo.irma.nz</nl>
+    </IssueURL>
+    <Attributes>
+        <Attribute id="number">
+            <Name>
+                <en>NHI number</en>
+                <nl></nl>
+            </Name>
+            <Description>
+                <en>NHI number</en>
+                <nl></nl>
+            </Description>
+        </Attribute>
```

Comment:
Maybe would be also useful to have `<Attribute revocation="true" />`. What do you think?

Reply:
I don't think NHI number is revocable!

Reply:
An NHI cannot be cancelled or invalidated? For those cases, I think they could be revoked, as the NHI is no longer valid and accurate. What do you think?

Reply:
@camfindlay Do you think NHI number is revocable?

Reply:
I can think of revocation for for duplicated NHIs or modernization of the NHI format. But we can leave this as non-revocable and mention this feature if needed.